apidescription.xml

```diff
@@ -38,8 +38,8 @@
         <apigee:tags>
           <apigee:tag primary="true">Opportunities</apigee:tag>
         </apigee:tags>
-        <apigee:authentication required="false"/>
-        <apigee:example url="/opportunity/version/3/do/query?user_key={user_key}&amp;api_key={api_key}"/>
+        <apigee:authentication required="true"/>
```

Comment:
Not sure if there is a way for us to test these changes since Apigee has replaced WADL with OpenAPI/Swagger.

Reply:
I reverted the apidescription.xml changes to avoid breaking any possible link. The code is outdated anyway, and if we find it is needed, we can open a bug to update that.

Reply:
Okay, sounds good. 👍